src/ray/protobuf/runtime_env_agent.proto

```diff
@@ -21,6 +21,8 @@ import "src/ray/protobuf/agent_manager.proto";
 message CreateRuntimeEnvRequest {
   string serialized_runtime_env = 1;
   bytes job_id = 2;
+  // Allocated resource instances.
```

Comment:
more descriptive comment about what this dictionary contains please

Reply:
Thanks for your suggestion. I have add some comment for it , PTAL.